addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -217,9 +221,11 @@
                                                 <input id="recruitment6" type="file"
                                                     class="form-control s_website_form_input o_resume_input"
                                                     name="Resume"/>
+                                                <span class="text-muted" style="font-size: 0.8rem">Optional if you provided a Linkedin profile</span>
                                             </div>
                                         </div>
-                                        <span class="text-muted" style="margin-left: 200px; font-size: 0.8rem">The resume is optional if you have a Linkedin profile</span>
```

Comment:
> just change it instead of hiding it and don't forget to update .pot file as well :)

Now i update pot file.

If we don't hide then we have to write a media query. I did it from media query before. [link](https://github.com/odoo-dev/odoo/commit/8a0ea0acd6cf173d8d71c855e1e3dfe958c8d1b5#diff-b11ebcefda69d7273ed42aef4054f51cba393463a6802a3935748ed7f685b53fR67-R73). what do you think ?